BowlingInJs.js

```diff
@@ -0,0 +1,51 @@
+class BowlingGame{
+    constructor(){
+        this.runningScore = 0;
+        this.rolls =[];
+    };
+    roll(pins){
+        this.rolls.push(pins);
+
+    };
+
+    get score() {
+        let score = 0;
+        let rollIndex = 0;
+
+        for (let frameIndex = 0; frameIndex < 10; frameIndex++) {
```

Comment:
So does this mean the frame index, if at all queried, would always be one behind what you'd expect (ie the first frame is frame 0, whereas in reality people would expect it to be 1?)